core/templates/dev/head/pages/collection_player/collection-player.module.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
2015 -> 2019

Reply:
Can I replace all 201* with 2018 in the module files? that will be faster and the modules are written in 2019.

Reply:
done

Reply:
done

Reply:
Yes, I think you can replace years with 2019 in all module files.